variation/query.py

```diff
@@ -114,7 +116,9 @@ def to_vrs(self, q: str)\
 
         if not translations:
             if q and q.strip():
-                translations = [Text(definition=q)]
+                text = models.Text(definition=q)
```

Comment:
I don't know if this is an actionable comment here, but is it weird that we have Text classes in both vrs-python and vrsatile-pydantic?

Reply:
Oh it definitely is weird and it felt weird doing it. I needed to use vrs-python in order to create the digest, but in the response models we use the pydantic models. I think maybe I should create an issue to create the digests in vrsatile.pydantic?